src/core/Jalasoft.TeamUp.Projects.Core/Validators/ProjectValidator.cs

```diff
@@ -0,0 +1,45 @@
+﻿namespace Jalasoft.TeamUp.Projects.Core.Validators
+{
+    using System;
+    using FluentValidation;
+    using Jalasoft.TeamUp.Projects.Models;
+
+    public class ProjectValidator : AbstractValidator<Project>
+    {
+        public ProjectValidator()
+        {
+            this.RuleFor(project => project.Id)
+                .NotEmpty().NotNull();
+
+            this.RuleFor(project => project.Name)
+                .Length(3, 70)
+                .Matches("^[a-zñ A-ZÑ]+$")
```

Comment:
I'm interested in this, what is the built-in validation message it returns? please double check this message is human readable for the consumer

Reply:
It is eligible for humans I
![ReturnThis](https://user-images.githubusercontent.com/85975279/128250179-05b5b669-7f92-4265-b3b2-2e315520d774.PNG)
 review it and I will send you a picture of what it responds